### PR TITLE
cli-docs: add runtime upgrade instructions

### LINKED ISSUE #1962: Add dock and runtime binary instructions to CLI getting started guide
Currently the instructions for dock and runtime upgrades are in the operators guide where it's not very easy to find them.

Since these two things are common actions and cause a lot of incoming support requests, I think it's useful to have them in the getting started guide.

Most CLI users won't need to know much more than how to install/boot urbit and upgrade their runtime. Other things they might have to know are in the cloud hosting instructions.

### PR DESCRIPTION
**Pull request by zalberico**
Fixes: https://github.com/urbit/urbit.org/issues/1962

This commit clarifies language around the Urbit runtime. I also add instructions about how to upgrade the runtime via next and if the user is not docked, how to dock.

### CODE REVIEW

File: content/getting-started/cli.md [hunk 1]
```diff
@@ -244,8 +246,41 @@ Whichever address and port it says there is the one to open in the browser.
...
+> default to port 8080):
+>
+> ```shell
+> sudo apt-get install libcap2-bin
+> sudo setcap 'cap_net_bind_service=+ep' ./urbit
+> ```
+
+If you've been running Urbit for a while (from before runtime version 1.9) 
+and these `.run` commands don't work for you, it probably means you need to dock 
+your pier. You can do this with the following command:
```
Comment (jalehman) [inline content/getting-started/cli.md]: What is docking your pier? Is this documented anywhere (e.g. by running `./urbit dock` w/o arguments? It's the first question that popped out to me. Probably wouldn't hurt to have something somewhere that explains what this is doing.

**Review by jalehman — COMMENTED** [review]
This looks great, tyfs.

File: content/getting-started/cli.md (hunk 1 above)
Comment (zalberico) [reply content/getting-started/cli.md]: https://operators.urbit.org/manual/running/vere#dock - I can link the word dock to this.

Comment (jalehman) [reply content/getting-started/cli.md]: Perfect, exactly what I was looking for.

**Review by jalehman — APPROVED** [review]

**Pull request merged**